src/utils/server/index.ts

```diff
@@ -196,6 +200,7 @@ export const OpenAIStream = async ({
 
       const parser = createParser(onParse);
 
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       for await (const chunk of res.body as any) {
```

Comment:
if(res.body) {
        for await (const chunk of res.body) {
          if (isFinished) {
            return;
          }
          parser.feed(decoder.decode(chunk as Buffer));
        }
      }